test/valid-data/type-typeof-function/main.ts

```diff
@@ -0,0 +1,3 @@
+import { ConnectionOptions } from "node:tls";
+
+export interface Test extends ConnectionOptions {}
```

Comment:
Please define a smaller test case.

Reply:
I liked the fact that it is really life example